src/index.html

```diff
@@ -6,8 +6,53 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__img"></div>
+      <p class="card__label">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__wrapper card__wrapper--stars">
+        <div class="card__stars">
+          <div class="card__star card__star--gold"></div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
```
it's must be the the BEM block and move all related styles to separate file